### PR TITLE
Add wagtail

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

**Comment by helloreuben** [comment]
Need these 7 packages built first

Willow >=0.4,<0.5 can use willow=0.4 - new build
django-modelcluster >=3.1,<4.0 can use django-modelcluster=3.1 - new build
django-taggit >=0.20,<1.0 can use django-taggit 0.22.1 - update needed
django-treebeard >=3.0,<5.0 can use django-treebeard 4.1.2 - new build
flake8-blind-except ==0.1.1 - new build
pyenchant ==1.6.8 -newbuild
sphinxcontrib-spelling >=2.3.0 can use django-treebeard =2.3.0 - newbuild

**Comment by sannykr** [comment]
@helloreuben : Thanks. I will add them to the list.

**Comment by sannykr** [comment]
@dougalsutherland : I can't figure out why there is an exception. I have built all dependencies. Could you please look at it and suggest something.

**Comment by sannykr** [comment]
@synapticarbors : Could you please review and merge.

File: recipes/wagtail/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```
Comment (synapticarbors) [inline recipes/wagtail/meta.yaml]: Please add an `entry_points` section for:

https://github.com/wagtail/wagtail/blob/master/setup.py#L101-L103

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

File: recipes/wagtail/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    - django-taggit >=0.20,<1.0
+    - django-treebeard >=3.0,<5.0
+    - djangorestframework >=3.1.3
+    - beautifulsoup4 >=4.5.1
+    - html5lib >=0.999,<1
+    - unidecode >=0.04.14
+    - willow >=0.4,<0.5
+    - requests >=2.11.1,<3.0
+   
+test:
```
Comment (synapticarbors) [inline recipes/wagtail/meta.yaml]: Please test the entry point:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands

**Comment by sannykr** [comment]
@synapticarbors : Thanks for the review. I have made the suggested changes.

**Comment by sannykr** [comment]
@synapticarbors : Could you please review and merge if everything looks okay.

**Pull request merged**